src/frontend/sharedComponents/PracticeMode.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { useContext } from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { View, StyleSheet, Text } from "react-native";
+import MaterialIcons from "react-native-vector-icons/MaterialIcons";
+import ConfigContext from "../context/ConfigContext";
+
+import { DARK_ORANGE, WHITE } from "../lib/styles";
+
+const m = defineMessages({
+  title: {
+    id: "sharedComponents.PracticeMode.title",
+    defaultMessage: "Practice Mode",
+    description: "Indicated to user that they are on practice mode",
+  },
+});
+
+interface PracticeModeProps {
+  children: React.ReactNode;
+}
```

Comment:
nit but can reduce this to use/extend `React.PropsWithChildren`

Reply:
Hmm, Im not sure I know the difference

Reply:
there isn't much but `React.PropsWithChildren` is basically the definition you have there but built into React's type defs

Reply:
Right, so this needs a typing, should it be `any` Like the child here can be anything as its simply a ui wrapper, no logic involved

Reply:
yeah in that case you can do something like:

```tsx
const SomeComponent = ({ children }: React.PropsWithChildren<{}>) => { ... } 
```

Reply:
Decent reference for understanding the various approaches to the same goal: https://www.newline.co/@bespoyasov/how-to-define-props-with-children-in-react-typescript-app--56bd18be